lens.cabal

```diff
@@ -348,6 +335,8 @@ library
 
   default-language: Haskell2010
 
+  x-docspec-extra-packages: simple-reflect deepseq
```

Comment:
currently it's commaless list. But now as I see it, it looks weird. Should we have commas?

Reply:
I'm fine with the way the current formatting.